extends Node2D

var level2 = {
	"question1" = "Is this a redox reaction?",
	"question2" = "What reactant was oxidized?",
	"question3" = "What reactant was reduced?",
	
	"1" = {
		"chemicalEquation": "ğ¶ğ»2ğ‘‚(ğ‘ğ‘) + ğ‘‚2(ğ‘”) â†’ ğ¶ğ‘‚2(ğ‘”) + ğ»2ğ‘‚(ğ‘™)",
		"redox": "Yes",
		"oxidized": "CH2O",
		"reduced": "O2",
		"incorrect1": "",
		"incorrect2": ""
	},
	
	"2" = {
		"chemicalEquation": "2ğ¾ğ¶8ğ»5ğ‘‚4 + 30ğ‘€ğ‘›2(ğ‘†ğ‘‚4)3 + 24ğ»2ğ‘‚ â†’ 16ğ¶ğ‘‚2 + 60ğ‘€ğ‘›ğ‘†ğ‘‚4 + 28ğ»2ğ‘†ğ‘‚4 + 2ğ¾ğ»ğ‘†ğ‘‚4",
		"redox": "Yes",
		"oxidized": "KC8H5O4",
		"reduced": "Mn2(SO4)3",
		"incorrect1": "",
		"incorrect2": ""
	},
	
	"3" = {
		"chemicalEquation": "2ğ¾ğ¶8ğ»5ğ‘‚4 + 10ğ¾2ğ¶ğ‘Ÿ2ğ‘‚7 + 41ğ»2ğ‘†ğ‘‚4 â†’ 16ğ¶ğ‘‚2 + 46ğ»2ğ‘‚ + 10ğ¶ğ‘Ÿ2(ğ‘†ğ‘‚4)3 + 11ğ¾2ğ‘†ğ‘‚4",
		"redox": "Yes",
		"oxidized": "KC8H5O4",
		"reduced": "K2Cr2O7",
		"incorrect1": "",
		"incorrect2": ""
	},
	
	"4" = {
		"chemicalEquation": "4ğ¶ğ‘™âˆ’(ğ‘ğ‘) + ğ‘‚2(ğ‘”) + 4ğ»+(ğ‘ğ‘) â†’ 2ğ»2ğ‘‚(ğ‘™) + 2ğ¶ğ‘™2(ğ‘”)",
		"redox": "Yes",
		"oxidized": "Cl-",
		"reduced": "O2",
		"incorrect1": "",
		"incorrect2": ""
	},
	
	"5" = {
		"chemicalEquation": "4ğ¹ğ‘’ğ‘†2(ğ‘ ) + 15ğ‘‚2(ğ‘”) + 2ğ»2ğ‘‚(ğ‘™) â†’ ğ‘ğ‘ğ‘ğ‘¡ğ‘’ğ‘Ÿğ‘–ğ‘ â†’ 4ğ¹ğ‘’3+ + 8ğ‘†ğ‘‚3-4 + 4ğ»+ + 2ğ¹ğ‘’2(ğ‘†ğ‘‚4)3 + 2ğ»2ğ‘†ğ‘‚4",
		"redox": "Yes",
		"oxidized": "FeS2",
		"reduced": "FeS2 and O2",
		"incorrect1": "",
		"incorrect2": ""
	},
	
	"6" = {
		"chemicalEquation": "2ğ¶ğ»2ğ‘‚(ğ‘ğ‘) â†’ ğ‘ğ‘ğ‘ğ‘¡ğ‘’ğ‘Ÿğ‘–ğ‘ â†’ ğ¶ğ»4(ğ‘”)+ ğ¶ğ‘‚2(ğ‘”)",
		"redox": "Yes",
		"oxidized": "Some of the carbon in CH2O",
		"reduced": "Some of the carbon in CH2O",
		"incorrect1": "",
		"incorrect2": ""
	},
	
	"7" = {
		"chemicalEquation": "ğ‘2(ğ‘”)+ ğ‘‚2(ğ‘”) â†’ â„ğ‘’ğ‘ğ‘¡ â†’ 2ğ‘ğ‘‚(ğ‘”)",
		"redox": "Yes",
		"oxidized": "N2",
		"reduced": "O2",
		"incorrect1": "",
		"incorrect2": ""
	},
	
	"8" = {
		"chemicalEquation": "2ğ‘2(ğ‘”) + 5ğ‘‚2(ğ‘”) + 2ğ»2ğ‘‚(ğ‘™) â†’ 4ğ»ğ‘ğ‘‚3(ğ‘ğ‘)",
		"redox": "Yes",
		"oxidized": "N2",
		"reduced": "O2",
		"incorrect1": "",
		"incorrect2": ""
	},
	
	"9" = {
		"chemicalEquation": "2ğ‘†(ğ‘ ) + 3ğ‘‚2(ğ‘”) + 2ğ»2ğ‘‚(ğ‘™) â†’ 2ğ»2ğ‘†ğ‘‚4(ğ‘ğ‘)",
		"redox": "Yes",
		"oxidized": "S (s)",
		"reduced": "O2",
		"incorrect1": "",
		"incorrect2": ""
	},
	
	"10" = {
		"chemicalEquation": "ğ¶ğ»4(ğ‘”) + 2ğ‘‚2(ğ‘”) â†’ ğ¶ğ‘‚2 + 2ğ»2ğ‘‚",
		"redox": "Yes",
		"oxidized": "Carbon in CH4",
		"reduced": "O2",
		"incorrect1": "",
		"incorrect2": ""
	},
	
	"11" = {
		"chemicalEquation": "6ğ¶ğ‘‚2(ğ‘”) + 6ğ»2ğ‘‚(ğ‘™) â†’ ğ‘â„ğ‘™ğ‘œğ‘Ÿğ‘œğ‘â„ğ‘¦ğ‘™ğ‘™ ğ‘ğ‘›ğ‘‘ ğ‘†ğ‘¢ğ‘›ğ‘™ğ‘–ğ‘”â„ğ‘¡ â†’ ğ¶6ğ»12ğ‘‚6(ğ‘ ) + 6ğ‘‚2(ğ‘”)",
		"redox": "Yes",
		"oxidized": "Oxygen in CO2 and in H20",
		"reduced": "Carbon in CO2",
		"incorrect1": "",
		"incorrect2": ""
	},
	
	"12" = {
		"chemicalEquation": "2ğ»ğ¶ğ‘™ğ‘‚(ğ‘ğ‘) + ğ¹ğ‘’(ğ‘ ) + 2ğ»+(ğ‘ğ‘) â†’ ğ¹ğ‘’2+(ğ‘ğ‘) + ğ¶ğ‘™2(ğ‘”) + 2ğ»2ğ‘‚(ğ‘™)",
		"redox": "Yes",
		"oxidized": "Fe (s)",
		"reduced": "Chlorine in HClO",
		"incorrect1": "",
		"incorrect2": ""
	},
	
	"13" = {
		"chemicalEquation": "2ğ»ğ¶ğ‘™ğ‘‚(ğ‘ğ‘) + ğ‘ƒğ‘(ğ‘ ) + 2ğ»+ â†’ ğ‘ƒğ‘2+(ğ‘ğ‘) + ğ¶ğ‘™2(ğ‘”) + 2ğ»2ğ‘‚",
		"redox": "Yes",
		"oxidized": "Pb (s)",
		"reduced": "Chlorine in HClO",
		"incorrect1": "",
		"incorrect2": ""
	},
	
	"14" = {
		"chemicalEquation": "2ğ¹ğ‘’2+(ğ‘ğ‘) + 1/2ğ‘‚2(ğ‘”) + 2ğ»+(ğ‘ğ‘) â†’ 2ğ¹ğ‘’3+(ğ‘ğ‘) + ğ»2ğ‘‚(ğ‘™)",
		"redox": "Yes",
		"oxidized": "Fe2+",
		"reduced": "O2",
		"incorrect1": "",
		"incorrect2": ""
	},
	
	"15" = {
		"chemicalEquation": "ğ¶ğ‘‘ğ¶ğ‘™2(ğ‘ğ‘) + ğ‘ğ‘2ğ‘†(ğ‘ğ‘) â†’ ğ¶ğ‘‘ğ‘†(ğ‘ ) + 2ğ‘ğ‘ğ¶ğ‘™(ğ‘ğ‘)",
		"redox": "No",
		"oxidized": "",
		"reduced": "",
		"incorrect1": "",
		"incorrect2": ""
	},
	
	"16" = {
		"chemicalEquation": "2ğ‘ğ‘›ğ‘†(ğ‘ ) + 2ğ»ğ¶ğ‘™(ğ‘ğ‘) â†’ ğ‘ğ‘›ğ¶ğ‘™2(ğ‘ğ‘) + ğ»2ğ‘†(ğ‘”)",
		"redox": "No",
		"oxidized": "",
		"reduced": "",
		"incorrect1": "",
		"incorrect2": ""
	}
}

# Called when the node enters the scene tree for the first time.
func _ready():
	pass # Replace with function body.


# Called every frame. 'delta' is the elapsed time since the previous frame.
func _process(delta):
	pass
